source/fundamentals/typescript.txt

```diff
@@ -245,7 +245,7 @@ of insert operations. The following table describes how different
 If you must specify the ``_id`` field as required in the type you define to represent
 documents in your collection but you do not want to specify values for the
 ``_id`` field in insert operations, use the ``OptionalId`` helper type when you
-create your collection. The ``OptionalId`` type accepts a type parameter as an
+create your myColl. The ``OptionalId`` type accepts a type parameter as an
```

Comment:
I: I think this is still easier to understand as "collection", even though the variable name is `myColl` (maybe a Find + Replace mistake?)

```suggestion
create your collection. The ``OptionalId`` type accepts a type parameter as an
```

Reply:
yup -- replace error